Add ability to import all files from a directory to import files dialog

Czer0xx opened the pull request:
<img width="2672" height="1112" alt="Screenshot 2025-07-28 at 13 05 03" src="https://github.com/user-attachments/assets/4b7b2624-966b-4522-91eb-8ed37bf3c77b" />

---

Czer0xx committed:
Added 'Import all' button in 'Maintenance/Import files', that imports all osu files from specified directory

bdach commented on osu.Game/Screens/Import/FileImportScreen.cs:
```diff
@@ -160,5 +204,41 @@ private void startImport(string path)
+            if (filesToImport.Count <= 0)
+                return;
+
+            Task.Factory.StartNew(async () =>
+            {
+                await game.Import(filesToImport.ToArray()).ConfigureAwait(false);
+
+                // Refresh the filepicker after importing
+                Schedule(() =>
+                {
+                    fileSelector.CurrentPath.TriggerChange();
+                });
+            }, TaskCreationOptions.LongRunning);
```
this is basically `startImport()` but inlined for some reason - just change the signature of `startImport()` to accept `params string[]` instead and use that here rather than copy-paste it here

bdach commented on osu.Game/Screens/Import/FileImportScreen.cs:
```diff
@@ -131,10 +148,37 @@ public override bool OnExiting(ScreenExitEvent e)
+
+            try
+            {
+                foreach (FileInfo file in directoryInfo.EnumerateFiles())
+                {
+                    if (game.HandledExtensions.Contains(file.Extension))
+                    {
+                        importAllButton.Enabled.Value = true;
+                        break;
+                    }
+                }
```
I would have probably LINQ'd this, I don't see a world where this is performance-critical, but I guess it's fine

Czer0xx committed:
Applied changes from review

 - Removed the try/catch block. (It was throwing an error only on Linux,
   when opened a symlink to directory to which user doesn't have access,
   idk if i should keep it or not.)
 - Replaced foreach loops with LINQ queries.
 - Replaced the copy-pasted startImport function with a call to it and
   changed its parameter type to "params string[]".

Czer0xx committed:
Removed unused directive, Replaced ".Where().Any()" with single call to Any().

Czer0xx committed:
Merge branch 'master' into import-all-button

bdach committed:
Improve condition (and surrounding commentary)

bdach committed:
Fix broken layout & use better copy
